language/move-lang/src/parser/syntax.rs

```diff
@@ -1481,8 +1517,8 @@ fn parse_parameter(tokens: &mut Lexer<'_>) -> Result<(Var, Type), Error> {
 
 // Parse a struct definition:
 //      StructDefinition =
-//          <DocComments> "resource"? "struct" <StructDefName> "{" Comma<FieldAnnot> "}"
-//          | <DocComments> "native" "resource"? "struct" <StructDefName> ";"
+//          <DocComments> <Ability>* "struct" <StructDefName> "{" Comma<FieldAnnot> "}"
```

Comment:
I'm guessing this is a holdover from an earlier draft of the PR with different syntax, right? The resource keyword should be removed but the `<Ability>*` should be replaced by the `("has" Comma<Ability>)?` syntax after the name.